src/main/java/domain/LadderResult.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LadderResult {
+
+    public static final int ONE_STEP = 1;
+    private Map<Integer, Integer> firstAndLastPosition;
+
+    public LadderResult(Ladder ladder, int participantsCount) {
+        initPosition(ladder, participantsCount);
+        calculatePosition(0, ladder, participantsCount);
+    }
+
+    private void initPosition(Ladder ladder, int participantsCount) {
+        this.firstAndLastPosition = IntStream.range(0, participantsCount)
+                .boxed()
+                .collect(Collectors.toMap(i -> i, i -> i));
```

Comment:
Function 구현체엔 identity()라고 하는, 인자를 그대로 제공하는 메서드를 제공합니다
```suggestion
        this.firstAndLastPosition = IntStream.range(0, participantsCount)
                .boxed()
                .collect(Collectors.toMap(Function.identity(),Function.identity()));
```

static import를 활용하면 더욱 축약할 수 있습니다
```suggestion
    private void initPosition(Ladder ladder, int participantsCount) {
        this.firstAndLastPosition = IntStream.range(0, participantsCount)
                .boxed()
                .collect(toMap(identity(), identity()));
    }
```